draft-iyengar-quic-delayed-ack.md

```diff
@@ -156,6 +156,12 @@ TRANSPORT_PARAMETER_ERROR. Note that while the endpoint's max_ack_delay
 transport parameter is in milliseconds (Section 18.2 of {{QUIC-TRANSPORT}}),
 min_ack_delay is specified in microseconds.
 
+The min_ack_delay transport parameter is a unilateral indication of support for
+receiving ACK_FREQUENCY frames.  The transport parameter is not required to be
+sent by both endpoints.  If an endpoint sends the transport parameter, the peer
```

Comment:
I'd drop this sentence, since it's not quite saying what you want to say at the moment and I think the paragraph works without it.

```suggestion
receiving ACK_FREQUENCY frames.  If an endpoint sends the transport parameter, the peer
```